tools/run_danesfield.py

```diff
@@ -517,6 +521,7 @@ def main(args):
     run_step(run_metrics_outdir,
              'run-metrics',
              cmd_args)
+    """
```

Comment:
Seems like adding another boolean flag to the configuration to set whether or not metrics should be run makes sense.

Reply:
Added as a command line argument (--run_metrics)